openedx/core/djangoapps/content/block_structure/api.py

```diff
@@ -0,0 +1,59 @@
+"""
+Contains some functions that had been defined in lms/djangoapps/course_blocks/api.py, but are actually
+needed in both lms and cms.
```

Comment:
Rewrite this docstring to explain what this module does, rather than why it was changed.  In three years, nobody's going to care that it once lived somewhere else.